integration-testing/election-manager/Makefile

```
@@ -0,0 +1,31 @@
+ELECTION_MANAGER := ../../apps/election-manager
+SMARTCARDS := ../../apps/module-smartcards
+SEMS := ../../apps/module-converter-ms-sems
+export PIPENV_VENV_IN_PROJECT=1
+export MODULE_SCAN_WORKSPACE=/tmp
+export MOCK_SCANNER_FILES=,,
+
+clean-downloads:
+	rm cypress/downloads/*
+
+build-election-manager:
+	make -C $(ELECTION_MANAGER) build; \
```

Unnecessary continuation? Or keeping it to make it easier to add another line?

```suggestion
	make -C $(ELECTION_MANAGER) build
```